topics/variant-analysis/tutorials/microbial-variants/tutorial.md

```diff
@@ -0,0 +1,235 @@
+
+The Galaxy wrapper for Snippy has the ability to change some of the underlying tool settings in the advanced section but it is not recommended.
+
+More can be read about SNP calling [here](https://en.wikipedia.org/wiki/SNV_calling_from_NGS_data)
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Section 1: Get the data
+
+The data for today is a subset of a real dataset from a Staphylococcus aureus bacteria.
+We have a closed genome sequence and an annotation for our "wildtype" strain.
+We have used a whole genome shotgun approach to produce a set of short sequence reads on an Illumina DNA sequencing instrument for our mutant strain.
+
+- The reads are paired-end
+- Each read is on average 150 bases
+- The reads would cover the original wildtype genome to a depth of 19x
+
+The files we will be using are:
+
+- mutant_R1.fastq & mutant_R2.fastq - the read files in fastq format.
+- wildtype.fna - The sequence of the reference strain in fasta format.
+- wildtype.gbk - The reference strain with gene and other annotations in genbank format.
+- wildtype.gff - The reference strain with gene and other annotations in gff3 format.
+
+This data is available at Zenodo using the following [link](https://doi.org/10.5281/zenodo.582600).
+
+> ### {% icon hands_on %} Hands-on: Get the data
+>
+> 1.  Import all of the following files into a new history:
+>     - [mutant_R1.fastq](https://zenodo.org/record/582600/files/mutant_R1.fastq)
+>     - [mutant_R2.fastq](https://zenodo.org/record/582600/files/mutant_R2.fastq)
+>     - [wiltype.fna](https://zenodo.org/record/582600/files/wildtype.fna)  
+>     - [wiltype.gbk](https://zenodo.org/record/582600/files/wildtype.gbk)
+>     - [wiltype.gff](https://zenodo.org/record/582600/files/wildtype.gff)
+>
+>    > ### {% icon tip %} Tip: Importing data via links
+>    >
+>    > * Copy the link location
+>    > * Open the Galaxy Upload Manager
+>    > * Select **Paste/Fetch Data**
+>    > * Paste the link into the text field
+>    > * Press **Start**
+>    {: .tip}
+>
+{: .hands_on}
+
+Your history pane should look like this after you have uploaded all of the data:
+
+![micro-start-history.png](../../images/micro-start-history.png)
+
+# Section 2: Find variants with Snippy
+
+We will now run the Snippy tool on our reads, comparing it to the reference. If we give Snippy an annotated reference, it will silently run a tool called snpeff which will figure out the effect of any changes on the genes and other features. If we just give Snippy the reference sequence alone without the annotations, it will not run snpeff.
+
+We have an annotated reference and so will use it in this case.
+
+> ### {% icon hands_on %} Hands-on: Run Snippy
+>
+> 1. Go to the Galaxy tools panel, and use the search box at the top to search for **Snippy** and then open it's interface.
+> 1. For **Reference type** select *Genbank*.
```

Comment:
We are trying to uniformize the parameter definition for the tool and we would like also to avoid the screenshot (more work to update when the tool changes and less accessible).
What do you think about something like:

```
> ### {% icon hands_on %} Hands-on: Run Snippy
>
> 1. Search for **Snippy** {% icon tool %} and run it with the following parameters
>   - "Reference type" to `Genbank`
>   - "Reference Genbank" to the `wildtype.gbk` file
>   - "Single or Paired-end reads" to `Paired`
>   - "Select first set of reads" to `mutant_R1.fastq`
>   - "Select second set of reads" to `mutant_R2.fastq`
>   - "Cleanup the non-snp output files" to `No`
{: .hands_on}
```